boskos/resources.yaml

```diff
@@ -42,8 +42,8 @@ resources:
     content: |
       gcp-perf-test:
         - clusters:
-          - machinetype: n1-highmem-8
-            numnodes: 7
+          - machinetype: n1-standard-32
+            numnodes: 4
```

Comment:
You want to allocate more CPU and Memory for perf cluster?

Reply:
agree, do we need this much for just running two pods? Seems a bit overkill unless we are going to do the servicegraph testing.

If its for servce graph testing, do we want 2 of these clusters?

Reply:
To clarify, gcp-perf-test currently has 2 clusters, and it doesn't autoscale

Reply:
@carolynhu the machine types are/may be different and I have never run anything on n1-highmem. 
We can reduce it to 2 nodes I already reduced it to 4. 
@howardjohn what change are you looking for?

Reply:
Just cost saving. Do we need 128 cores when we are only using a small amount? We are just running two pods right?